torchtune/modules/attention.py

```diff
@@ -10,6 +10,85 @@
 import torch
 from torch import nn, Tensor
 from torchtune.modules.kv_cache import KVCache
+from torchtune.utils._version import torch_version_ge
+from torchtune.utils.attention_bias import _MaskType
+from torchtune.utils.logging import get_logger, log_once
+
+_log: logging.Logger = get_logger()
+
+
+if torch_version_ge("2.5.0"):
+    from torch.nn.attention.flex_attention import BlockMask, flex_attention
```

Comment:
Can you clarify: If the user doesnt have torch>=2.5, they cannot use block causal. Is that right?

If the answer is yes, then i think we need to express it a bit better in this if/else block